src/components/AccountForm.vue

```diff
@@ -694,7 +694,15 @@ h4 {
 .flex-row {
 	display: flex;
 }
-
+@-moz-document url-prefix() {
```

Comment:
could you find out why firefox styles it differently?

Reply:
we have dedicated styles for firefox in other parts of our code. I dont know though, why specifically these buttons are broken only for FF.